src/client/components/TableNodeExt/CellNodeExt/CellNodeExt.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react'
+
+import DataColumn from 'client/components/DataGrid/DataColumn'
+import Select from 'client/components/Select'
+import { Option } from 'client/components/Select/types'
+import { ColumnNodeExt } from 'client/components/TableNodeExt/types'
+import TextInput from 'client/components/TextInput'
+
+type Props = {
+  datum: Record<string, any>
+  column: ColumnNodeExt
+  onChange: (uuid: string, colName: string, value: any) => void
+  disabled: boolean
+}
+
+type CellProps = {
+  value: string | Array<string>
+  onChange: (newValue: string | Option) => void
+  disabled: boolean
+  options?: Array<any>
+  column: ColumnNodeExt
+}
+
+const components: Record<string, React.FC<CellProps>> = {
+  text: (props) => (
```

Comment:
This is only function because the underlaying component is a class component and we would need to refactor it
Should we create a new component instead?

Reply:
Create inner component that wraps that
src/client/components/TableNodeExt/CellNodeExt/TextInput
src/client/components/TableNodeExt/CellNodeExt/Select